### DISCUSSION TITLE
Legacy docs

### DISCUSSION THREAD

**Discussion by stefanofa** [rainbow-me/rainbowkit#1255] [Feedback]
Where can I find the legacy docs pre v1.0.0?

I think you have to make an effort to make it easier to find because anyone forced to continue using ethers will have a hard time looking for it. This also concerns the wagmi documentation.

**Comment by DanielSinclair** [comment]
Legacy docs for 0.12.x are now available here: https://legacy-rainbowkit.vercel.app/docs/installation
Wagmi docs for 0.12.x are also here: https://0.12.x.wagmi.sh/
